react/src/js/components/Consonant/Cards/Card.jsx

```diff
@@ -561,7 +562,21 @@ const Card = (props) => {
                     {iconAlt}
                 </span>
                 }
-                { highlightedTitle ? (
+                { (accessibilityHeading && highlightedTitle) &&

```

Comment:
Minor nitpick recommendation: Rename `accessibilityHeading` to `isTitleOnly` for clarity.